go.mod

```diff
@@ -11,7 +11,7 @@ require (
 	github.com/gregjones/httpcache v0.0.0-20180305231024-9cad4c3443a7 // indirect
 	github.com/imdario/mergo v0.3.6 // indirect
 	github.com/json-iterator/go v1.1.5 // indirect
-	github.com/martin-helmich/go-varnish-client v0.1.0
+	github.com/martin-helmich/go-varnish-client v0.1.1-0.20200302071804-9c9194146dfe
```

Comment:
Reminder to self: This is a direct checkout of the dependent PR martin-helmich/go-varnish-client#5. Merge that one and update the dependence at this place again, before merging this one.